package.json

```
@@ -77,12 +77,19 @@
         "readme": "evalmd README.md",
         "postlint": "eclint check $(git ls-files | xargs find 2> /dev/null | grep -vE 'node_modules|\\.git' | grep -v dist/)",
         "lint": "eslint --ext=js,mjs .",
+        "benchmark": "cd benchmark && npm install && node index.js",
+        "benchmark:parse": "cd benchmark && npm install && node index.js --parse",
+        "benchmark:stringify": "cd benchmark && npm install && node index.js --stringify",
+        "benchmark:baseline": "cd benchmark && npm install && node index.js --save-baseline",
+        "benchmark:compare": "cd benchmark && npm install && node index.js --baseline benchmark/baseline.json",
```

```suggestion
        "benchmark": "cd benchmark && npm install && node .",
        "benchmark:parse": "cd benchmark && npm install && node . --parse",
        "benchmark:stringify": "cd benchmark && npm install && node . --stringify",
        "benchmark:baseline": "cd benchmark && npm install && node . --save-baseline",
        "benchmark:compare": "cd benchmark && npm install && node . --baseline benchmark/baseline.json",
```

also instead of all these `:foo` scripts, can these be `npm run benchmark -- --parse` etc?